pittgoogle/pubsub.py

```diff
@@ -356,6 +379,8 @@ class Subscription:
         ),
     )
     schema_name: str | None = attrs.field(default=None)
+    attribute_filter: Optional[str] = attrs.field(default=None)
+    udf: Optional[str] = attrs.field(default=None)
```

Comment:
I think these should just be kwargs in the `touch()` function. These are only used once when creating the subscription. And when a user creates a `Subscription` object for a gcp subscription that already exists, our code doesn't make the filter or udf available in the `Subscription` object even if they exist on the actual subscription. So I don't think they should be class attributes.